# msg to set PID target point

uint8 DoF

float32 Target
